app/views/pages/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>New Page</h1>
+
+<%= form_for @page do |f| %>
+  <div class="form-group">
+    <%= f.label :title %>
+    <%= f.text_field :title, class: 'form-control', placeholder: "Enter page title" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :body %>
+    <%= f.text_area :body, class: 'form-control', placeholder: "Enter page body" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.submit "Save", class: 'btn btn-success' %>
+  </div>
+
+<% end %>
```

Comment:
Make sure to right-click-Inspect this form in the browser, to refresh your mind on what gets generated by the `form_for` helper. Specifically, look at the `<form>` tag and its attribute, such as `action`. Check whether you understand the complete page creation workflow, from browser (pages/new) to backend (`PagesController#create`) and then back to the browser (/pages/:id).